_src/pldoc/static/js/size-slider.js

```diff
@@ -22,38 +24,30 @@ $(function() {
         },
 
         listenForSlider: function() {
-            var that = this;
-
-            that.vars.sliderControl.on('input change', function() {
+            IconFontSliderControl.vars.sliderControl.on('input change', function(event) {
                 // oninput - chrome, firefox, safari
                 // onchange - ie9-11
-                that.updateIconSize($(this).val());
-                that.updateAria($(this).val());
-                that.updateInputText($(this).val());
+                // console.log(event.target.value);
+                IconFontSliderControl.updateIconSize(event.target.value);
+                IconFontSliderControl.updateAria(event.target.value);
+                IconFontSliderControl.updateInputText(event.target.value);
```

Comment:
You use event.target.value 3 times so it would make sense to set it as a var at the top of the function and reference that instead.